test/Layer2LibVCTest.js

```diff
@@ -123,7 +123,7 @@ async function test(redisClient) {
 
   let lBob = new Layer2lib("https://rinkeby.infura.io", optionsBob)
 
-  web3.setProvider(new web3.providers.HttpProvider('http://localhost:8545'))
+  web3.setProvider(new web3.providers.HttpProvider('http://localhost:7545'))
```

Comment:
Is the port 7545 your local nodes port? i'd prefer to assume a testnet port for now rather than everyone is running this version of the client

Reply:
Changed to rinkeby